service/package/gem2rpm.yml

```diff
@@ -9,16 +9,6 @@
   # "msgfmt" tool
   BuildRequires:  gettext-runtime
   Requires:       dbus-1-common
```

Comment:
is it still required here for gem or only needed for new package with files?

Reply:
Good catch! Also, I will need to make the new package require the older one